test_runner/src/main/kotlin/ftl/reports/outcome/CreateMatrixOutcomeSummary.kt

```diff
@@ -2,15 +2,14 @@ package ftl.reports.outcome
 
 import com.google.api.services.toolresults.model.Environment
 
-fun TestOutcomeContext.createMatrixOutcomeSummary(): Pair<BillableMinutes, List<TestOutcome>> =
-    steps.calculateAndroidBillableMinutes(projectId, testTimeout) to
-        if (environments.hasOutcome())
-            environments.createMatrixOutcomeSummaryUsingEnvironments()
-        else {
-            if (steps.isEmpty()) println("No test results found, something went wrong. Try re-running the tests.")
-            steps.createMatrixOutcomeSummaryUsingSteps()
-        }
+fun TestOutcomeContext.createMatrixOutcomeSummary() = Pair(
+    steps.calculateAndroidBillableMinutes(projectId, testTimeout),
+    if (environments.hasOutcome())
+        createMatrixOutcomeSummaryUsingEnvironments()
+    else {
+        if (steps.isEmpty()) println("No test results found, something went wrong. Try re-running the tests.")
+        createMatrixOutcomeSummaryUsingSteps()
+    }
+)
```

Comment:
for consistency
```suggestion

    steps.calculateAndroidBillableMinutes(projectId, testTimeout) to
    if (environments.hasOutcome())
        createMatrixOutcomeSummaryUsingEnvironments()
    else {
        if (steps.isEmpty()) println("No test results found, something went wrong. Try re-running the tests.")
        createMatrixOutcomeSummaryUsingSteps()
    }

```

Reply:
Yeah, I changed to `Pair(...)` because leads to nasty indention
```kotlin
steps.calculateAndroidBillableMinutes(projectId, testTimeout) to
            if (environments.hasOutcome())
                createMatrixOutcomeSummaryUsingEnvironments()
            else {
                if (steps.isEmpty()) println("No test results found, something went wrong. Try re-running the tests.")
                createMatrixOutcomeSummaryUsingSteps()
            }
```
And IMHO it's less readable. `if` looks like it is chained, `to` is very easy to omitted.
I don't have a strong opinion regarding this and sure I can change it if you prefer that way :)

Reply:
On the second thought, I decided to refactor this, even more, please check latest implementation

Reply:
like it!